.github/workflows/release-drafter-ca.yml

```diff
@@ -0,0 +1,147 @@
+name: Schedule a release and tag image for the cotnainer app
+on:
+  workflow_dispatch:
+  schedule:
+     - cron: "0 0 * * 3"
+
+env:
+  RELEASEPREFIX: auditlogca-
+  VERSIONPREFIX: v
+  REGISTRY: ghcr.io
+  REPOSITORY: altinn/altinn-auth-audit-log
+
+jobs:     
+  getlatestrelease:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      deployments: write
+      issues: write
+      packages: write
+    
+    steps:
+      - name: Get latest version for function app
+        uses: oprypin/find-latest-tag@v1
+        continue-on-error: true
+        with:
+          repository: ${{ env.REPOSITORY }}
+          releases-only: true
+          prefix: ${{ env.RELEASEPREFIX }}${{ env.VERSIONPREFIX }}  # Other kinds of tags are irrelevant;
+        id: auditlogca_version
+      - name: setup current year
+        id: get_year
+        run: |
+          echo "currentyear=$(date +%Y)" >> $GITHUB_ENV
+
+      - name: print status
+        run: "echo Release could not be found. Request failed with status '${{ steps.get_latest_release_version.outputs.status }}'"
+      
+      - name: split release version f.ex v2023.1 will be split into output0 v2023 and output1 1
```

Comment:
This seems like a very convoluted way to do some simple string manipulation. Also, I'd like to have a discussion regarding this versioning scheme.